core/templates/pages/topic-editor-page/modal-templates/questions-opportunities-select-difficulty-modal.component.html

```diff
@@ -39,10 +39,19 @@ <h3 class="oppia-modal-header" *ngIf="linkedSkillsWithDifficulty.length > 0">
             (click)="startQuestionCreation()"> Continue
     </button>
   </div>
+</div>
+
+<div>
   <div *ngIf="linkedSkillsWithDifficulty.length === 0">
-    <button class="btn btn-secondary" (click)="goToSelectSkillView()"> Back </button>
+    <div class="loading-screen">
```

Comment:
@Alchong1 Can we use the existing [loading-message component](https://github.com/oppia/oppia/blob/bbb778e89ad66f9206480be9aa85cf47f607e56b/core/templates/base-components/loading-message.component.ts#L26)?

Reply:
Thanks for the suggestion, I hadn't tried using that component before. However, I don't think we can use the exist loading-message component as it currently is because it is for displaying a full page loading message. I tried using it instead for the loading screen and wasn't able to get it to appear at all, possibly due to the clashes with the popup background.

Here is a change I tried using the component:
![image](https://user-images.githubusercontent.com/112645984/233158546-193c8913-5467-4fc4-b6b5-f45fa8a2ad4a.png)

Here is the change visualized on the development server:

https://user-images.githubusercontent.com/112645984/233161339-0a2b04b6-7c89-4afd-8d87-f9ec826eef2a.mp4

Here is the change in Slow 3G:

https://user-images.githubusercontent.com/112645984/233161479-a8c748ac-7b5a-4027-8d16-8a3866dd2c57.mp4


I'm not very familiar with components so it's possible I'm not implementing it correctly, but I do think using a component might be a step in the right direction.